### PR TITLE
feat(kit): add pulse component

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/kit/components/pulse/pulse.component.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    styleUrls: ['./pulse.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    animations: [tuiFadeIn, tuiScaleIn],
+})
+export class TuiPulseComponent {
+    @Input()
+    public playing = true;
+
+    @Output()
+    public readonly playingChange = new EventEmitter<boolean>();
```
Comment (waterplea) [inline projects/kit/components/pulse/pulse.component.ts:30]: Unnecessary, it does not change state internally.

File: projects/kit/components/pulse/pulse.component.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    @HostBinding('@tuiFadeIn')
+    @HostBinding('@tuiScaleIn')
+    protected readonly animation = tuiToAnimationOptions(inject(TUI_ANIMATIONS_SPEED));
+
+    @HostBinding('style.--t-animation-state')
```
Comment (waterplea) [inline projects/kit/components/pulse/pulse.component.ts:36]: Should we move this and animations to `host` to avoid `@HostBinding` maybe?

**Review by waterplea — APPROVED** [review]

File: projects/kit/components/pulse/pulse.component.ts (hunk 2 above)
Comment (splincode) [reply projects/kit/components/pulse/pulse.component.ts]: fixed

File: projects/kit/components/pulse/pulse.component.ts (hunk 1 above)
Comment (splincode) [reply projects/kit/components/pulse/pulse.component.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**